packages/ui/src/ui/libs/monaco-yql-languages/clickhouse/clickhouse.contribution.ts

```diff
@@ -7,24 +7,46 @@ import {
 } from '../_.contribution';
 import {languages} from 'monaco-editor';
 import {createProvideSuggestionsFunction} from '../helpers/createProvideSuggestionsFunction';
-import {parseClickHouseQuery} from '@gravity-ui/websql-autocomplete';
-import {generateClickhouseAdditionalSuggestion} from './clickhouse.keywords';
+import {
+    generateClickhouseAdditionalSuggestion,
+    generateClickhouseOldSafariSuggestions,
+} from './clickhouse.keywords';
 import {MonacoLanguage} from '../../../constants/monaco';
+import {Toaster} from '@gravity-ui/uikit';
+import {rumLogError} from '../../../rum/rum-counter';
 
 registerLanguage({
     id: MonacoLanguage.CHYT,
     extensions: [],
-    loader: () =>
-        import('./clickhouse').then((module) => {
+    loader: async () => {
+        const yql = await import('./clickhouse');
+        try {
+            //autocomplete breaks safari < 16.6
+            const autocomplete = await import('@gravity-ui/websql-autocomplete');
```

Comment:
It is good idea to provide `webpackChunkName` for the chunk, and probably we need a separate function for it:
```ts
function laodAutocompleteChunk() {
   try {
        await import(/* webpackChunkName: "websql-autocomplete" */ '@gravity-ui/websql-autocomplete');
   }
   catch(e) {
        ....
        return undefined;
   }
}
```